lib/src/slices/api_google/api_google_service.dart

```diff
@@ -3,67 +3,41 @@
  * MIT license. See LICENSE file in root directory.
  */
 
-import 'dart:convert';
-
-import 'package:googleapis/gmail/v1.dart';
-import 'package:googleapis_auth/googleapis_auth.dart' as gapis;
-import 'package:http/http.dart';
-import 'package:http/http.dart' as http;
 import 'package:logging/logging.dart';
 
 import '../../utils/helper_json.dart';
 import '../api_auth_service/api_auth_service.dart';
 import '../api_auth_service/model/api_auth_service_account_model.dart';
-import '../api_email_msg/model/api_email_msg_model.dart';
-import '../api_email_sender/model/api_email_sender_model.dart';
-import '../data_bkg/model/data_bkg_model_page.dart';
+import '../data_bkg/data_bkg_service_provider.dart';
 import '../info_carousel_card/model/info_carousel_card_model.dart';
+import 'model/api_google_model.dart';
 import 'repository/api_google_repository_info.dart';
 
-class ApiGoogleService {
+class ApiGoogleService implements DataBkgServiceProvInterface {
```

Comment:
Shouldn't this implement both DataBkgServiceProviderInterface and DataBkgServiceEmailInterface?

Reply:
nevermind. ignore this. I see that you split the service into 2 files